lib/twine/cli.rb

```diff
@@ -306,8 +306,13 @@ def self.prepare_description!(options, summary_width)
         end
       end
 
+      if IO.console
+        console_width = IO.console.winsize[1] if IO.console
+      else
```

Comment:
160 feels like a large default. Is 100 way too small given the help output twine generates?